jenkins/debian/debos/scripts/buster-ltp.sh

```diff
@@ -39,8 +39,8 @@ echo '  ]}' >> $BUILDFILE
 
 git clone --depth=1 -b 20200515 ${LTP_URL}
 cd ltp && make autotools 
-./configure 
-make all
+./configure --with-open-posix-testsuite
+make -j`nproc` all
```

Comment:
This might explain why LTP rootfs builds were taking so long, if it was all single-threaded.

It might have deserved a separate commit, but that's fine.

Reply:
Your build took 33min, so I think it's much better that before - in addition to fixing the problem for timers tests.